docs/geedocs/answer/6078762.html

```diff
@@ -67,7 +67,7 @@
         <tr>
           <td>17725523</td>
           <td>When attempting to install GEE 5.1.0 on RHEL 6 using the GUI installer, the process is likely to fail.</td>
```

Comment:
Should we be documenting how to install GEE 5.1 in the GEE open source documentation?